lang/mosml/Portfile

```diff
@@ -1,51 +1,97 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
 
-name                mosml
-version             2.01
-revision            2
+github.setup        kfl mosml 13c581aec46eea134e478f2e2b6456278e36ecce
```

Comment:
as usual, this should take a released version, potentially with some patches on top.

Reply:
The last released version was from 2014: https://github.com/kfl/mosml/tags
Let us stay with commit-based setup, please.

Reply:
> The last released version was from 2014: [kfl/mosml/tags](https://github.com/kfl/mosml/tags) Let us stay with commit-based setup, please.

Fine with me